core/src/main/java/com/linecorp/armeria/server/ServiceConfigSetters.java

```diff
@@ -157,4 +159,23 @@ ServiceConfigSetters blockingTaskExecutor(ScheduledExecutorService blockingTaskE
      */
     @UnstableApi
     ServiceConfigSetters multipartUploadsLocation(Path multipartUploadsLocation);
+
+     /**
+      * Sets an {@link EventLoopGroup worker group} to be used when executing service codes.
```

Comment:
```suggestion
      * Sets a {@linkplain EventLoopGroup worker group} to be used when executing service codes.
```
(because it's rendered as `a worker group`)